OpenRA.Game/WorldUtils.cs

```diff
@@ -56,9 +56,7 @@ public static bool HasVoice(this Actor a, string voice)
 
 		public static SoundInfo GetVoices(this Actor a)
 		{
-			var selectable = a.Info.Traits.GetOrDefault<SelectableInfo>();
-			if (selectable == null) return null;
-			var v = selectable.Voice;
+			var v = a.TraitOrDefault<IVoiced>().VoiceSet;
```

Comment:
You need to keep the null check here because `TraitOrDefault` can give you null, and you have no guarantees that whoever calls this method will check beforehand.